test/var_compatibility_test/test_host.F90

```diff
@@ -361,11 +361,12 @@ program test
         'flag_indicating_cloud_microphysics_has_graupel            ',         &
         'flag_indicating_cloud_microphysics_has_ice                '/)
 
-   character(len=cm), target :: test_outvars1(7) = (/                         &
+   character(len=cm), target :: test_outvars1(8) = (/                         &
         'ccpp_error_code                                           ',         &
         'ccpp_error_message                                        ',         &
         'effective_radius_of_stratiform_cloud_ice_particle         ',         &
         'effective_radius_of_stratiform_cloud_liquid_water_particle',         &
+        'effective_radius_of_stratiform_cloud_rain_particle        ',         &
```

Comment:
general question about this test - with the new additions that you've provided, are we testing that the unit conversions are being correctly applied (e.g. the results are correct?)

Reply:
Correct. These four schemes include the scenarios in the original issue.
![Screenshot 2024-10-30 at 9 15 51 AM](https://github.com/user-attachments/assets/e5d5748c-4594-492e-bdb3-328067c507ee)
![Screenshot 2024-10-30 at 9 16 29 AM](https://github.com/user-attachments/assets/e1453f5a-f906-458d-ba98-5c15673f323e)
![Screenshot 2024-10-30 at 9 16 36 AM](https://github.com/user-attachments/assets/06abcd78-88aa-417e-9b49-65755b1fca5e)
![Screenshot 2024-10-30 at 9 16 42 AM](https://github.com/user-attachments/assets/73545c8e-c16c-4333-83fc-2a157c52939d)
![Screenshot 2024-10-30 at 9 16 50 AM](https://github.com/user-attachments/assets/339ccd03-4b5e-4ffc-81eb-292531e30aea)